docs/docs/getting-started/installation.md

```diff
@@ -1,131 +1,84 @@
-- [Jaeger](https://www.jaegertracing.io/) instance, if selected.
-- [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), if selected.
-- [PokeShop demo app](https://github.com/kubeshop/pokeshop/), if selected.
+```bash
+tracetest server install
+```
 
-### Kubernetes
+:::note
+Follow the prompts and continue with all the default settings.
+This will generate an empty `docker-compose.yaml` file and a `./tracetest/` directory that contains another `docker-compose.yaml` and two more config files. One for Tracetest and one for OpenTelemetry collector.
+:::
 
-You can run Tracetest in a Kubernetes cluster. This setup is ideal for a CI/CD environment, QA teams working on shared environments, etc.
-You can use a remote or local (minikube, etc) cluster. We'll even help you setup a local cluster, if you need one.
+Start Docker Compose from the directory where you ran `tracetest server install`.
 
-![Installer using Kubernetes](../img/installer/1_kubernetes_0.7.0.png)
+```bash
+docker compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up -d
+```
 
-**Tools required (installed if missing)**:
-- kubectl
-- helm
+```bash title="Condensed expected output from the Tracetest container:"
+Starting tracetest ...
+...
+2022/11/28 18:24:09 HTTP Server started
+...
+```
 
-If you selected to run locally and want the installer to set up [minikube](https://minikube.sigs.k8s.io/docs/) for you:
-- Docker
+Open your browser on [`http://localhost:11633`](http://localhost:11633).
 
-**Requirements**:
-- Jaeger or other compatible backend. If missing, the installer will help you configure one.
-- OpenTelemetry Collector. If missing, the installer will help you configure one.
+Create a [test](../web-ui/creating-tests.md).
 
-**Optionals**:
-- [PokeShop demo app](https://github.com/kubeshop/pokeshop/)
+Read the detailed setup on the [CLI installation page](./cli).
 
-**Result**:
-- `tracetest` helm chart deployed in the `tracetest` (configurable) namespace.
-- [Jaeger](https://www.jaegertracing.io/) instance deployed in the `tracetest` namespace, if selected.
-- [Cert Manager](https://cert-manager.io/), if selected.
-- [Jaeger Operator](https://www.jaegertracing.io/docs/latest/operator/), if selected.
-- [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) deployed in the `tracetest` (configurable) namespace, if selected.
-- [PokeShop demo app](https://github.com/kubeshop/pokeshop/) deployed in the `demo` namespace, if selected.
+:::info
+Running a test againts `localhost` will resolve as the 127.0.0.1 inside the Tracetest container. To run tests againts apps running on your local machine, add them to the same network and use service name mapping instead. Example: Instead of running an app on `localhost:8080`, add it to your Docker Compose file, connect it to the same network as your Tracetest service, and use `service-name:8080` in the URL field when creating an app.
```

Comment:
Maybe it's worth saying that you can reach services running on your local machine using:

#### Linux (docker version < 20.10.0)
172.17.0.1:8080

#### MacOS (docker version >= 18.03) and Linux (docker version >= 20.10.0)
host.docker.internal:8080

Reply:
https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66